packages/builders/scripts/inter-protocol/price-feed-core.js

```diff
@@ -32,11 +32,16 @@ export const defaultProposalBuilder = async (
   assert(AGORIC_INSTANCE_NAME, 'AGORIC_INSTANCE_NAME is required');
   assert(Array.isArray(oracleAddresses), 'oracleAddresses array is required');
 
-  if (!brandIn) {
+  if (brandIn) {
```

Comment:
changing this file means putting another version of it (in a bundle) on the chain. Is that essential to do?

There are other clients... they probably won't be confused, because they refer to the code by bundleID. But I think we already have 2 versions of this on chain, and keeping them straight is hard enough.

I've been exploring a new contract to do this sort of thing.

 - https://github.com/agoric-labs/ag-power-tools/pull/8

Reply:
I reverted these changes.